lighthouse-core/test/gather/gather-runner-test.js

```diff
@@ -263,22 +270,27 @@ describe('GatherRunner', function() {
   });
 
   it('uses correct emulation form factor', async () => {
-    let emulationParams;
+    let deviceMetricsParams;
     const driver = getMockedEmulationDriver(
-      params => emulationParams = params,
+      params => deviceMetricsParams = params,
```

Comment:
might be easier to just use `jest.fn` here for the case it's asserting `not.toHaveCalled()`?

Reply:
done